src/GraphQL/Mutations/BaseAuthResolver.php

```diff
@@ -14,8 +14,8 @@ class BaseAuthResolver
      */
     public function buildCredentials(array $args = [], $grantType = "password")
     {
-        $args = collect($args);
-        $credentials = $args->except('directive')->toArray();
+        $credentials['username'] = $args['input']['username'];
```

Comment:
The reason why we do not use `input` here is because the schema has the spread directive.